x/authz/keeper/keeper_test.go

```diff
@@ -445,27 +445,6 @@ func (s *TestSuite) TestGetAuthorization() {
 	}
 }
 
-func (s *TestSuite) TestGetAuthorizations() {
```

Comment:
why was this removed?

Reply:
Reverting this PR (https://github.com/cosmos/cosmos-sdk/pull/17527), as we reverted in v0.47 as well (https://github.com/cosmos/cosmos-sdk/pull/17407#discussion_r1311486405).
The fix is good and valid, and I verified no one was using this method (https://github.com/cosmos/cosmos-sdk/pull/17334#issuecomment-1695225420) but just want to be cautious for closed source chains.
This is fixed in v0.50+ anyway.